docs/workforce-management-framework/topics/ref-introducing-raincatcher.adoc

```diff
@@ -0,0 +1,69 @@
+[id='ref-introducing-raincatcher-{chapter}']
+= {WFM-RC-NameShort} Reference Material
+
+This section includes:
+
+. xref:highlighting-raincatcher-areas-of-focus-{chapter}[Highlighting {WFM-RC-NameShort} Areas of Focus]
+. xref:describing-raincatcher-technically-{chapter}[Describing {WFM-RC-NameShort} Technically]
+. xref:identifying-raincatcher-repositories-{chapter}[Identifying {WFM-RC-NameShort} Repositories]
+. xref:differentiating-raincatcher-from-tech-preview-{chapter}[Differentiating {WFM-RC-NameShort} from 'Tech Preview']
+
+[id='highlighting-raincatcher-areas-of-focus-{chapter}']
+== Highlighting {WFM-RC-NameShort} Areas of Focus
+
+.Areas of Focus
+|===
+|Area |Focus
+
+|Framework
+|A generic and developer friendly framework (set of classes) for building mobile and website Workforce Management solutions
+
+|Modules
+|Modules specific to {WFM-RC-NameShort}, for example: signature-module, scheduler
+
+|Integrations
+|Integrations to satisfy {WFM-RC-NameShort} needs, for example: authentication, storage, offline capabilities, server side synchronization
+
+|User Interface
+|Graphical User Interfaces are implemented in AngularJS and Angular
+|===
+
+[id='describing-raincatcher-technically-{chapter}']
+== Describing {WFM-RC-NameShort} Technically
+
+* link:https://nodejs.org/en/[Node.js] used for server side code
+* Hybrid mobile development using https://cordova.apache.org/[Cordova]
+* Object Oriented approach (classes and interfaces)
+* Focus on module reuse and flexibility
+* Incorporates open-source technologies, for example: MongoDb, Keycloak, Passport.js
+
+[id='identifying-raincatcher-repositories-{chapter}']
+== Identifying {WFM-RC-NameShort} Repositories
+ 
+The {WFM-RC-NameShort} is available in 3 github repositories:
```

Comment:
You are correct (see style guide for more info) - updated:

"⁠numbers
Spell out numbers zero through nine, any number that begins a sentence, a number that precedes another number (four 6-pound bags; eleven 20-pound bags), approximations (thousands of…), and very large values, (4 billion). Use numerals for numbers 10 and greater, negative numbers, fractions, percentages, decimals, measurements, references to book sections (Chapter 3, Table 5, Page 11), and numbers less than 10 if they appear in the same paragraph as numbers of 10 or greater (You answered 8 out of 14 questions correctly). Also use numerals when referring to registers (such as R1), code (such as x = 6), and release versions (Red Hat Enterprise Linux 6, Source-Navigator 4.5)."